students/Matt_Hudgins/Lesson05/mailroom_3.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python3
+
+# chmod +x mailroom.py needs to be performed before executable
+
+'''
+    File Name: mailroom_3.py
+    Author: Matt Hudgins
+    Date created: 5/20/18
+    Date last modified: 5/20/18
+    Python Version 3.6.4
+'''
+import sys
+from collections import deque
+
+# This is the donor list
+donors = {}
+donors['Rick Grimes'] = [5.00, 10.00, 2.00]
+donors['Shane Walsh'] = [4.00, 10.00, 9.00]
+donors['Carl Grimes'] = [72.00, 10.00, 88.00]
+donors['Morgan Jones'] = [68.00, 10.00, 98.00]
+
+
+def donor_list():
+    '''This function prints a list of all current donors'''
+    for i in donors:
+        print(i)
+    return
+
+
+def adding_a_donor(newname):
+    '''This function adds a name to the donor list'''
+    donors[newname] = []
+    adding_a_donation(newname)
+    return
+
+
+def adding_a_donation(newname):
+    '''This function adds a donation to the the list'''
+    donation_amount = round(float(input(f'Enter a Donation amount for\n'
+    '{newname}\n')), 2)
+    if newname in donors.keys():
+        donors[newname].append(donation_amount)
+    elif newname not in donors:
+        donors[newname] = [donation_amount]
+    print(donors)
+    return
+
+
+def send_letter(newname, amount):
+        letter = 'Dear {},\n Thank you for your donation of ${}'
+        print(letter.format(newname, amount))
+        print('\n')
+        return
```

Comment:
These bare `return` statements are not necessary. Python functions will return `None` by default if no return value is specified and the convention is to leave them off the function if you don't need them for anything.